src/main/resources/page/js/dummy-content-displayer.js

```diff
@@ -0,0 +1,7 @@
+class WriteOutDataOnDummyPage {  
```

Comment:
The class should represent one specific thing, but this class name is too specific. (This name should be a function)

DummyPageContentPopulator? Or DummyPageBuilder? Or DummyPageDataPopulator? (For me the first one is the best)

Reply:
Okay, I will rename it, thanks!